doc/source/dev/contributor/development_workflow.rst

```diff
@@ -214,15 +214,28 @@ been added to ``upstream`` that affect your work. In this case, follow the
 Writing the commit message
 --------------------------
 
-Commit messages should be clear and follow a few basic rules.  Example::
+Commit messages should be clear and follow a few basic rules.
 
-   ENH: add functionality X to SciPy.<submodule>.
+Example::
+
+   MAINT/TST: fft: remove xp backend skips, test `fftfreq` `device`
 
    The first line of the commit message starts with a capitalized acronym
-   (options listed below) indicating what type of commit this is. Then a blank
-   line, then more text if needed.  Lines shouldn't be longer than 72
-   characters.  If the commit is related to a ticket, indicate that with
-   "See #3456", "See ticket 3456", "Closes #3456", or similar.
+   (or multiple, options listed below) indicating what type of commit this is.
+   Then a blank line, then more text if needed.
+   References to code names should be enclosed in backticks.
+   If changes are limited to certain submodules or functions, they should be
+   included after the acronym(s) - backticks are not needed here.
+
+Example::
+
+   BUG:sparse.linalg.gmres: add early exit when `x0` already solves problem
```

Comment:
minor: missing a space after `BUG:`

Reply:
this was intentional, to accommodate for @ilayn's preference in the linked issue

Reply:
Okay, let's leave it as is then. It looks a little off to me; if a next contributor wants to tweak it then let's just delete the second example rather than bikeshed over a very minor point.